### ISSUE TITLE
Data Augmentation

### ISSUE THREAD

**Issue by rafariva** [jiny2001/dcscn-super-resolution#32]
Hi, i have my dataset with a lot of images .PNG, 640x480 with Bit depth 16. when i run augmentation.py, its transform all of then in Bit Depth 8, and i "loosing data" (my image become almost white). one more time, no clue.

**Comment by jiny2001** [comment]
Hi, really sorry about it. I have never thought there could be 16-bit PNG. And the library I'm using doesn't handle it correctly...

BTW, the code doesn't delete original data. It doesn't transform but it creates new data based on the original. So you still keep original data, right?

Best,

**Comment by rafariva** [comment]
Yes. I still keep original data.. Its ok. But, i can do data augmentation... This Image are in 16 bits. I was trying to fix the code... Not lucky

**Comment by jiny2001** [comment]
Well, for image scaling, converting and loading, we have to be very careful.
Some library has a bad performance on scaling, or they automatically changes value range in some condition.

I'll check it later on this weekend. thx.

**Comment by jiny2001** [comment]
Hi,
Sorry, I tried some image libraries but had no luck. Gave up to deal with 16 bit PNG.

I found someone is saying PIL.Image on python3 would work for 16bit PNG. And I tried it but it didn't work.

Even if I could read 16bit PNG, I must carry the flag that shows original image is 16 bit and modify for loading/scaling/converting/evaluation/generation parts. It seems like too much to me for now.

Best,
